scripts/package.json

```diff
@@ -122,8 +121,8 @@
     "progress": "^2.0.3",
     "prompts": "^2.3.0",
     "puppeteer": "^1.13.0",
-    "raw-loader": "^0.5.1",
-    "react-hot-loader": "^4.12.15",
+    "raw-loader": "^4.0.2",
```

Comment:
IIRC raw-loader has some pretty dramatically different behavior between versions, maybe like whether it assigns to exports or uses a default export?